core/services/validators.go

```diff
@@ -438,6 +438,9 @@ func ValidatedOracleSpecToml(tomlString string) (spec offchainreporting.OracleSp
 	if err := validateTimingParameters(spec); err != nil {
 		return spec, err
 	}
+	if err := validateMonitoringURL(spec); err != nil {
```

Comment:
Why is the monitoring URL in the spec? Should be in the global config right? There is only ever one config at a time.

Reply:
I thought we did this deliberately to enable node operators to set multiple monitoring endpoints... Is that a mistake?

Reply:
Yup, so we want a per-job configuration option but the default is to fall back to the global one. Check latest changes

Reply:
Seems a little YAGNI to me and is going to require a rearchitect of how it all works behind here.